src/App.tsx

```diff
@@ -1,25 +1,57 @@
-import React from 'react';
+// import React, { useState } from 'react';
+import { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
+// import { event } from 'cypress/types/jquery';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [lengthItem, setLengthItem] = useState('5');
+  const [currentPagin, setCurrentPagin] = useState(1);
```

Comment:
try to choose more self descriptive naming. shorten currentPagin is not clear. It is better to use something like currentPage